jcl/src/java.base/share/classes/openj9/internal/foreign/abi/InternalDowncallHandler.java

```diff
@@ -791,44 +744,39 @@ Object runNativeMethod(Addressable downcallAddr, SegmentAllocator segmtAllocator
 			if (retStruSegmt == null) {
 				throw new OutOfMemoryError("Failed to allocate native memory for the returned memory segment");
 			}
-			/*[IF JAVA_SPEC_VERSION >= 20]*/
+			/*[IF JAVA_SPEC_VERSION >= 21]*/
 			retMemAddr = retStruSegmt.address();
-			/*[ELSE] JAVA_SPEC_VERSION >= 20 */
+			/*[ELSE] JAVA_SPEC_VERSION >= 21 */
 			retMemAddr = retStruSegmt.address().toRawLongValue();
-			/*[ENDIF] JAVA_SPEC_VERSION >= 20 */
+			/*[ENDIF] JAVA_SPEC_VERSION >= 21 */
 		}
 
 		/* Add the scopes of the downcall related memory addresses to the set
 		 * to ensure they are kept alive till the downcall is finished.
 		 */
-		/*[IF JAVA_SPEC_VERSION >= 20]*/
+		/*[IF JAVA_SPEC_VERSION >= 21]*/
 		addMemArgScope(downcallAddr.scope());
 		addMemArgScope(stateSegmt.scope());
-		/*[ELSE] JAVA_SPEC_VERSION >= 20 */
+		/*[ELSE] JAVA_SPEC_VERSION >= 21 */
 		addMemArgScope(downcallAddr.address().scope());
-		/*[ENDIF] JAVA_SPEC_VERSION >= 20 */
+		/*[ENDIF] JAVA_SPEC_VERSION >= 21 */
 
 		long returnVal = 0;
```

Comment:
The initial value here is redundant; please remove it.

Reply:
Removed.